src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -276,7 +276,7 @@ public async Task<IActionResult> Resume([FromServices] IScriptHostManager script
             {
                 ScriptHostState currentState = scriptHostManager.State;
 
-                _logger.LogDebug($"Received request to resume a draining host - host status: {currentState.ToString()}");
+                _logger.LogDebug("Received request to resume a draining host - host status: {currentState}", currentState.ToString());
```

Comment:
`.ToString()` calls should be removed. The log system will call `ToString()` only when the log is actually being written